### PR TITLE
Revise to use `#:unsafe` declaration.

### PR DESCRIPTION
**Pull request by samth**

### CODE REVIEW

**Comment by mflatt** [comment]
I think you didn't mean to leave the future visualizer in use here.

File: pkgs/racket-benchmarks/tests/racket/benchmarks/shootout/spectralnorm-par.rkt [hunk 1]
```diff
@@ -4,11 +4,8 @@
 ;; Translated from Mike Pall's Lua version.
 ;; Parallelized by Sam Tobin-Hochstadt
 
-(require racket/cmdline racket/future
-         racket/require (for-syntax racket/base)
-         (filtered-in (λ (name) (regexp-replace #rx"unsafe-" name ""))
-                      racket/unsafe/ops)
-         (only-in racket/flonum make-flvector))
+(require racket/cmdline racket/future racket/fixnum racket/flonum future-visualizer)
```
Comment (mflatt) [inline pkgs/racket-benchmarks/tests/racket/benchmarks/shootout/spectralnorm-par.rkt:7]: Is the change to require `future-visualizer` intended?

File: pkgs/racket-benchmarks/tests/racket/benchmarks/shootout/spectralnorm-par.rkt [hunk 2]
```diff
@@ -18,44 +15,46 @@
...
+            y i
+            (let L ([a 0.0] [j 0])
+              (if (fx= j N) a
+                  (L (fl+ a (fl* (flvector-ref x j) (A j i)))
+                     (fx+ j 1)))))))
+(define (AtAv x y t N) (Av x t N) (Atv t y N))
+(define u (make-flvector N 1.0))
+(define v (make-flvector N))
+(define t (make-flvector N))
+(visualize-futures
```
Comment (mflatt) [inline pkgs/racket-benchmarks/tests/racket/benchmarks/shootout/spectralnorm-par.rkt:47]: Along the same lines, was this use of `visualize-futures` meant to be part of the commit?

File: pkgs/racket-benchmarks/tests/racket/benchmarks/shootout/spectralnorm-par.rkt (hunk 1 above)
Comment (samth) [reply pkgs/racket-benchmarks/tests/racket/benchmarks/shootout/spectralnorm-par.rkt]: Whoops, fixed.

**Comment by mflatt** [comment]
LGTM

**Pull request merged**